package-lock.json

```diff
@@ -3602,15 +3601,13 @@
           "version": "1.0.0",
           "resolved": false,
           "integrity": "sha1-ibTRmasr7kneFk6gK4nORi1xt2c=",
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
I think those changes are not desired.

It looks as if `package-lock.json` was regenerated by older version of npm

Reply:
I hate these changes so much.

Maybe it was me. In every project we've been using a `package-lock.json` file we've changed that thing back- and force. Everyone produced a different file. EVERY. SINGLE. TIME.

Reply:
Remind me again why we actually version `package-lock.json`?
- It doesn't serve local development (IIRC we tend to have different setups, and there's no agreement we should follow `package-lock.json`)
- It doesn't serve global installation (npm ignores that)

What does it serve?

Reply:
Yes, true. I guess we just version this because `npm` says so when it generates it. But yes, it really serves no purpose right now.

Reply:
Ok, I'll probably propose a PR that removes it then.